JSON.md

```diff
@@ -0,0 +1,77 @@
+Here is a sample of JSON output when nothing is detected:
+
+    {
+        "video-rate": 33.3, 
+        "drpai-rate": 2.5, 
+        "detections": []
+    }
+
+Here is a sample of JSON output of Yolo models when the tracking is **off**:
+
+    {
+        "video-rate": 32.3, 
+        "drpai-rate": 2.5, 
+        "detections": [
+            { 
+                "class": "person", 
+                "probability": 0.91, 
+                "box": {
+                    "centerX": 425, 
+                    "centerY": 325, 
+                    "width": 297, 
+                    "height": 282
+                }
+            },
+            {
+                "class": "wine glass", 
+                "probability": 0.52, 
+                "box": {
+                    "centerX": 87, 
+                    "centerY": 133, 
+                    "width": 122, 
+                    "height": 172
+                }
+            }
+        ]
+    }
+
+Here is a sample of JSON output of Yolo models when the tracking is **on**:
+
+    {
+        "video-rate": 32.3, 
+        "drpai-rate": 2.5, 
+        "detections": [
+            {
+                "id": 2, 
+                "seen_first": "Fri Dec 22 23:20:56 2023", 
+                "seen_last": "Fri Dec 22 23:21:46 2023", 
+                "class": "person", 
+                "probability": 0.91, 
+                "box": {
+                    "centerX": 425, 
+                    "centerY": 325, 
+                    "width": 297, 
+                    "height": 282
+                }
+            },
+            {
+                "id": 5, 
+                "seen_first": "Fri Dec 22 23:25:53 2023", 
+                "seen_last": "Fri Dec 22 23:26:07 2023", 
+                "class": "wine glass", 
+                "probability": 0.52, 
+                "box": {
+                    "centerX": 87, 
+                    "centerY": 133, 
+                    "width": 122, 
+                    "height": 172
+                }
+            }
+        ],
+        "tracked-count": {
+            "total": 6,
+            "person": 2,
+            "wine glass": 3,
+            "chair": 1
```

Comment:
How much sense does it make to have an extra section that just contains the counts? It's a fairly cheap compute operation to go through the JSON string above and count how many elements of each class there are in the string, right? It appears to be redudndant data. Is there a particular advantage for this?

Reply:
This is the historical data since the start. So this section says how many has been seen in total during the past "N" hours.

The rest of the JSON is the current data. So it says what objects are currently visible in front of the camera.

Reply:
> This is the historical data since the start. So this section says how many has been seen in total during the past "N" hours.

Ah okay, cool then! Makes sense. It would be good then to add another attribute that specifies the time of the accumulation window, right? And that time should be configurable too.

Reply:
@matin added the tracking window in this PR: https://github.com/MistySOM/gstreamer1.0-drpai/pull/25